projects/core/components/svg/svg.component.ts

```diff
@@ -112,11 +126,11 @@ export class TuiSvgComponent {
     }
 
     private get isUrl(): boolean {
-        return this.icon.endsWith(`.svg`);
+        return typeof this.icon === `string` ? this.icon.endsWith(`.svg`) : false;
```

Comment:
`return typeof this.icon === 'string' && this.icon.endsWith('.svg')`

Reply:
fixed